spikeextractors/extractors/neoextractors/neobaseextractor.py

```diff
@@ -0,0 +1,85 @@
+import numpy as np
+import neo
+
+from ..recordingextractor import RecordingExtractor
+from ..sortingextractor import SortingExtractor
+
+
+class NeoBaseRecordingExtractor(RecordingExtractor):
+    NeoRawIOClass = None
+    def __init__(self, **kargs):
+        self.neo_reader = self.NeoRawIOClass(**kkargs)
+        self.neo_reader.parse_header()
+        
+        # TODO propose a meachanisim to select the appropriate segment
+        # in case there are several
+        num_block = self.neo_reader.block_count()
```

Comment:
Can spikeextractors support either multiple blocks or segments through a MultiExtractor?

Reply:
Yes it could theorically could.
At the moment it is a bit too complicated.
And for now, no of existing extractor from file source have multi segment.

Reply:
I see. We currently take multiple extractors (segments) and combine them later with MultiExtractors. You are right.